blog/_posts/2018-07-09-IntroducingTeamRubies.md

```diff
@@ -0,0 +1,123 @@
+
```

Comment:
Hi @prateksha ! Thanks for the PR. Extra one line space at the start is unnecessary. Please remove it.